internal/services/scheduler/scheduler.go

```diff
@@ -0,0 +1,47 @@
+package scheduler
+
+import (
+	"time"
+
+	"go.uber.org/zap"
+	"gorm.io/gorm"
+
+	"github.com/al-kirpichenko/gofermart/cmd/gophermart/config"
+	"github.com/al-kirpichenko/gofermart/internal/api"
+	"github.com/al-kirpichenko/gofermart/internal/models"
+	"github.com/al-kirpichenko/gofermart/internal/services/accrual"
+)
+
+func UpdateOrders(s *api.Server) {
+
+	var orders []models.Order
+
+	for {
+
+		s.DB.Where("status = ?", "PROCESSING").Find(&orders).Limit(200)
```

Comment:
статус лучше вынести в константу
и кажется был второй нефинальный статус
лимит тоже было бы лушче вынести в конфиг, 200 кажется очень уж много
```suggestion
		s.DB.Where("status = ?", _statusProcessing).Find(&orders).Limit(config.ordersBatchNum)
```